minetest.conf.example

```diff
@@ -94,6 +94,8 @@
 #random_input = false
 #    Timeout for client to remove unused map data from memory
 #client_unload_unused_data_timeout = 600
+#    Maximum number of mapblocks for client to be kept in memory
+#client_mapblock_limit = 1000
```

Comment:
It should be documented that this count-based limit can be disabled by setting this to -1.